analyzer/src/funTest/kotlin/DotNetSupportTest.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2019 Bosch Software Innovations
+ * Based on:
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.analyzer
+
+import com.here.ort.model.Identifier
+import com.here.ort.model.OrtIssue
+import com.here.ort.model.PackageReference
+import com.here.ort.model.Scope
+import java.io.File
```

Comment:
Nit: `import java.io.File` should come after `import io.kotlintest.*` (to sort alphabetically), and we group imports by top-level package by using blank lines in between (similar in other files).